### PR TITLE
Creates automation events page

### PR DESCRIPTION
**Pull request by katjacksonWB**
## Description

Creates new automation events page.

## Ticket

[DOCS-920](https://wandb.atlassian.net/browse/DOCS-920)

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [ ] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [ ] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [ ] I merged the latest changes from `main` into my feature branch before submitting this PR.

[DOCS-920]: https://wandb.atlassian.net/browse/DOCS-920?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by ngrayluna — CHANGES_REQUESTED** [review]
Doc CI is failing

File: docs/guides/automations/events.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+displayed_sidebar: default
+---
+
+An *event* is a change that takes place in the W&B ecosystem. 
+
+## Model Registry
```
Comment (noaleetz) [inline docs/guides/automations/events.md]: Hey @katjacksonWB - plz take a look at my note [here](https://weightsandbiases.slack.com/archives/C05754EPT26/p1722377417289199?thread_ts=1722270837.161739&cid=C05754EPT26). These docs should be in the language of generic Registry. Automations can be added to any Registry. Can you review the Registry docs to make sure you're aligned on the changes and update the model registry specific language in here?

File: docs/guides/automations/events.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+The Model Registry supports two event types: **Linking a new artifact to a registered model** and **Adding a new alias to a version of the registered model**.
+
+See [Link a model version](./link-model-version.md) for information on how to link model versions and [Create a custom alias](../artifacts/create-a-custom-alias.md) for information an artifact aliases.
```
Comment (noaleetz) [inline docs/guides/automations/events.md]: let's break this down into more concrete sections for each event type. For each we should give a brief code snippet on how you trigger this event, and then link them to the docs you have here. But we should be more specific on what exactly needs to be done to trigger the event.

File: docs/guides/automations/events.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+
+
+:::tip
+Use the **Linking a new artifact to a registered model** event type to test new model candidates. Use the **Adding a new alias to a version of the registered model** event type to specify an alias that represents a special step of your workflow, like `deploy`, and any time a new model version has that alias applied.
+:::
+
+## Artifacts
```
Comment (noaleetz) [inline docs/guides/automations/events.md]: We should be clear here on the differences between this section and the Registry section. I would make the titles specific:
- Event Types for Automations applied to a Registry
- Event Types for Automations applied to Artifacts in a project

Let's make sure to link to the section in the outline on automations scope

File: docs/guides/automations/events.md
Comment (noaleetz) [inline docs/guides/automations/events.md:1]: Joe created these diagrams to help users visualizations the 2 events - can we add these to the docs? 
Screenshot 2024-08-02 at 4 20 11 PM
Screenshot 2024-08-02 at 4 20 22 PM

**Review by noaleetz — CHANGES_REQUESTED** [review]
v0 changes requested

**Comment by noaleetz** [comment]
Screenshot 2024-08-05 at 3 05 55 PM
Hey @katjacksonWB - please make sure to request changes when you're ready for another review otherwise I won't know when to take a look

**Pull request merged**